docs/03_tutorials/tic-tac-toe-game-contract.md

```diff
@@ -655,34 +654,22 @@ After the deployment and the transaction is confirmed, the contract is already a
 [[info | Test Account]]
 | If you have not created these accounts already, refer to this article for creating test accounts [Create Test Accounts](../02_development-environment/07_create-test-accounts.md)
 
-### Create
+### Create a game
 
 We are going to use `bob` and `alice` accounts to play this game:
 
 ```bash
 cleos push action tic.tac.toe create '{"challenger":"bob", "host":"alice"}' --permission alice@active
 ```
 
-### Move
+### Make moves
```

Comment:
I am not sure about this just bringing attention to it, did we agreed to have in titles all capitals? I think we did.

Reply:
Good to use Title-style Capitalization for headings.

### Create a Game
### Make a Move

And so on...